### PR TITLE
Feature/scss autosuggest refactor

### PR DESCRIPTION
**Pull request by lpsmitamsterdam**

### CODE REVIEW

File: src/app/components/Header/auto-suggest/AutoSuggest.tsx [hunk 1]
```diff
@@ -22,6 +22,41 @@ const NoResults = styled(Paragraph)`
...
+  }
+
+  @media screen and (min-width: 769px) {
+    max-width: calc(100% - 45px);
+  }
+`
+
+const AutoSuggestTip = styled.h3`
+  font-size: 14px; // was font-size($xs-font) (scaled)
+  color: ${themeColor('tint', 'level5')}; //was $secondary-gray60 !note not the same but closest;
```
Comment (lpsmitamsterdam) [inline src/app/components/Header/auto-suggest/AutoSuggest.tsx:55]: @chrisvanmook @thomasm0 how shall we approach the mismatch between colors in the sass theme and the asc? should we maybe refine this together with design?

Comment (chrisvanmook) [reply src/app/components/Header/auto-suggest/AutoSuggest.tsx:55]: as discussed, let's leave it for now, we can create another ticket and PR's to fix this and allign with design

File: src/app/components/Header/auto-suggest/AutoSuggestItem.tsx [hunk 2]
```diff
@@ -43,9 +43,29 @@ export interface AutoSuggestItemProps {
...
+  padding-bottom: calc(${10 / 2}px); // was $base-whitespace
+  padding-top: calc(${10 / 3}px); // was $base-whitespace
+  position: relative;
+  text-align: left;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+  text-decoration: none;
+  &:hover,
+  &--active {
```
Comment (chrisvanmook) [inline src/app/components/Header/auto-suggest/AutoSuggestItem.tsx:65]: is `--active` being used anywhere?

**Review by chrisvanmook — APPROVED** [review]

File: src/app/components/Header/auto-suggest/AutoSuggestItem.tsx (hunk 2 above)
Comment (lpsmitamsterdam) [reply src/app/components/Header/auto-suggest/AutoSuggestItem.tsx:65]: good you mention this actually, as useTraverselist & headersearch refer to the active class so i have to fix that as the classname has changed

File: src/app/components/Header/auto-suggest/AutoSuggest.tsx [hunk 3]
```diff
@@ -22,6 +22,41 @@ const NoResults = styled(Paragraph)`
...
+  position: absolute;
+  right: 45px; + 10px * 0.5;
+  max-height: calc(100vh - 160px); // 160 is max height of the header
+  overflow: auto;
+
+  // Makes sure the autosuggest box starts behind the SearchBar
+  top: 45px - 5px;
+
+  @media screen and ${breakpoint('min-width', 'tabletM')} {
+    top: 45px; // This is the height of the SearchBarMenu from @amsterdam/asc-ui
```
Comment (thomasm0) [inline src/app/components/Header/auto-suggest/AutoSuggest.tsx:42]: I noticed 5px more top spacing in this version than compared to production. On production this media-query only applies the styling `max-width: calc(100% - 45px)`

**Review by thomasm0 — CHANGES_REQUESTED** [review]
Everything looks good except for the AutoSuggestDropdown 45px instead of 40px

**Review by thomasm0 — APPROVED** [review]

**Pull request merged**